app/frontend/src/components/admin/Developer.vue

```diff
@@ -79,8 +81,8 @@ export default {
         this.apiRes = user.data;
       } catch (error) {
         this.addNotification({
-          message: 'Failed to get user from RBAC, see console',
-          consoleError: `Error getting User from RBAC: ${error.message}`,
+          message: this.message,
+          consoleError: this.consoleErrMsg + `: ${error.message}`,
```

Comment:
Why not use the direct translation here?

Reply:
I did not declare the variable. It was already declared. I just did the translation

Reply:
We should use the direct translation here.